lib/not-found.js

```diff
@@ -31,7 +31,8 @@ module.exports = options => {
 
     res.status(404).render('404', {
       title: translations.title,
-      description: translations.description
+      description: translations.description,
+      startLink: req.path.replace(/^\/([^\/]*).*$/, '$1')
```

Comment:
What is this doing?

Can I ask for either a comment or some more obvious code please?